lib/pharos/scripts/configure-essentials.sh

```diff
@@ -2,6 +2,19 @@
 
 set -e
 
+if [ "${SET_HTTP_PROXY}" = "true" ]; then
+    cat <<EOF >/etc/environment
+PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
```

Comment:
It would be nice to avoid hardcoding the `PATH`, but that would need some kind of `lineinfile` helper using sed instead...

Reply:
Fixed.